operations/boot_orchestration/Create_a_Session_Template_to_Boot_Compute_Nodes_with_CPS.md

```diff
@@ -1,61 +1,63 @@
 # Create a Session Template to Boot Compute Nodes with CPS
 
-When compute nodes are booted, the Content Projection Service \(CPS\) and Data Virtualization Service \(DVS\) project the root file system \(rootfs\) over the network to the compute nodes by default.
+When compute nodes are booted, the Content Projection Service \(CPS\) and Data Virtualization Service \(DVS\) project the root file system \(`rootfs`\) over the network to the compute nodes by default.
 
-Another option when compute nodes are booted is to download their rootfs into RAM.
+Another option when compute nodes are booted is to download their `rootfs` into RAM.
 
-### Procedure
+## Procedure
 
-1.  Use either the cray bos session create CLI command or the bash script to create a session template.
+1. Use the CLI to create a session template.
```

Comment:
I removed it because this wasn't about the CLI vs the API, it was about the CLI vs a bash script.
- No one is going to use a bash script to make these calls as it is presented here, not as a CLI alternative.
- This isn't the place to include things like how to get a token
- It clutters up this example and makes it difficult to read and find the relevant information.

Now if we want to talk about CLI vs API I think it would be reasonable to include the API equivalents of all the CLI calls, but stripped down without the "bash script" context.  That said, I still not sure this is the place for that.  This is an example use case and I feel like that should be as simple as possible.  The pages that cover all the possible calls are a better place for that information.